### ISSUE TITLE
[Bug]: Remote Node `lastOnline` API returns 404 while node test succeeds

### ISSUE THREAD

**Issue by Flylink** [MHSanaei/3x-ui#4966]
### Describe the bug

# Bug Report: Remote Node `lastOnline` API returns 404 while node test succeeds

## Version

* 3x-ui: 3.2.8
* Main node: 3.2.8
* Remote node: 3.2.8

## Description

A remote node can be added successfully and passes the connection test in the panel.

The remote panel is accessible through HTTPS and the configured base path.

However, the main panel continuously reports:

```text
WARNING - X-UI: remote-node lastOnline fetch failed:
POST panel/api/clients/lastOnline:
context deadline exceeded
```

## Environment

Main node:

* HTTPS enabled
* SQLite database

Remote node:

* HTTPS enabled
* SQLite database
* Fresh installation
* Same 3x-ui version as the main node

## What works

* Remote node can be added successfully.
* Connection test succeeds.
* HTTPS is reachable.
* TLS certificate is valid.
* Panel login works.
* Port is reachable from the main server.

## Manual verification

From the main server:

```bash
curl -vk -X POST \
https://remote-domain:remote-port/randomBasePath/panel/api/clients/lastOnline
```

Response:

```text
HTTP/1.1 404 Not Found
```

The request reaches the remote panel successfully and TLS handshake completes, but the endpoint returns 404.

## Expected behavior

The remote node should expose the API endpoint required by the main panel and return a valid response for:

```text
/panel/api/clients/lastOnline
```

or the main panel should not attempt to call a non-existing endpoint.

## Actual behavior

The remote node test succeeds, but the main panel continuously logs:

```text
lastOnline fetch failed
```

and manual requests to the endpoint return:

```text
404 Not Found
```

## Additional notes

* Reinstalling the remote node does not resolve the issue.
* Re-adding the node does not resolve the issue.
* HTTPS, certificates, firewall rules and connectivity have been verified.
* The issue appears to be related specifically to the `lastOnline` API endpoint.

### How to reproduce the problem

1. Install 3x-ui 3.2.8 on Server A (main node).

2. Install 3x-ui 3.2.8 on Server B (remote node).

3. Enable HTTPS on both nodes.

4. Configure a custom `webBasePath` on the remote node.

5. Add the remote node to the main node using the Remote Node feature.

6. Verify that:

   * The remote node connection test succeeds.
   * The remote panel is reachable via browser.
   * Login to the remote panel works.

7. Wait for the main node to start synchronizing client statistics.

8. Observe the log on the main node:

```text
WARNING - X-UI: remote-node lastOnline fetch failed:
POST panel/api/clients/lastOnline:
context deadline exceeded
```

9. Manually test the endpoint:

```bash
curl -vk -X POST \
https://remote-node:port/basePath/panel/api/clients/lastOnline
```

10. The endpoint returns:

```text
HTTP/1.1 404 Not Found
```

while the remote node itself remains accessible and functional.

### Relevant logs

```shell

```

### 3x-ui version

3.2.8

### How did you install 3x-ui?

install.sh script

### Operating system

Ubuntu 24

### Is the panel behind a reverse proxy or CDN?

None

### Before submitting

- [x] I searched [existing issues](../issues?q=is%3Aissue) and this bug has not been reported.
- [x] I am running the latest released version of 3x-ui (or have verified the bug still exists on it).
- [x] This bug report is written in English.
- [x] I have redacted any sensitive data (tokens, real domains, client UUIDs).

**Comment by Flylink** [comment]
ping avg = 40.7 ms
packet loss = 0%
curl time_total = 0.151s

**Comment by Flylink** [comment]
Additional diagnostic result:

From the main server, an unauthenticated request to the same endpoint returns immediately:

```bash
curl -vk -m 10 \
https://remote-domain:remote-port/basePath/panel/api/inbounds/list
```

Result:

```text
HTTP/1.1 404 Not Found
```

The response is immediate, TLS handshake succeeds, and the remote panel is reachable.

However, the background node sync job still logs:

```text
node traffic sync: fetch from remote-node failed:
GET panel/api/inbounds/list:
context deadline exceeded
```

So the network connection itself does not hang. The timeout appears to happen only during the authenticated API request made by the sync job.
@MHSanaei

**Comment by Flylink** [comment]
I tested with the real API token.

Both authenticated requests return 200 OK:

GET /panel/api/inbounds/list
POST /panel/api/clients/lastOnline

So the API token, endpoint, TLS and connectivity are valid.

However, the main panel sync job still logs:

POST /panel/api/clients/lastOnline: context deadline exceeded

The URL, port and basePath in the log are now correct.

This suggests the issue is inside the background sync job or its shared 4-second context budget, not in network connectivity or authentication.
